doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -380,6 +380,14 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-7-0
+
+KRKNWK-19443: The device commissioning window can be opened for longer than 900 seconds, what violates the Matter specification.
```

Comment:
Can we add information that the issue impacts following Matter Certification Test Cases:
[TC-CADMIN-1.21] Open commissioning window - durations max and max+1 [BCM] [DUT - Commissionee]
[TC-CADMIN-1.22] Open commissioning window - durations max and max+1 [ECM] [DUT - Commissionee]